php/Blockonomics.php

```diff
@@ -435,47 +447,10 @@ public function is_error_template($template_name) {
 
     // Adds the header to the blockonomics page
     public function load_blockonomics_header($template_name, $additional_script=NULL){
```

Comment:
Since we are not loading the header in this function anymore, we can rename load_blockonomics_header function accordingly.